input/postgres/relation_stats.go

```diff
@@ -51,7 +51,10 @@ SELECT c.oid,
 			 COALESCE(sio.toast_blks_read, 0),
 			 COALESCE(sio.toast_blks_hit, 0),
 			 COALESCE(sio.tidx_blks_read, 0),
-			 COALESCE(sio.tidx_blks_hit, 0)
+			 COALESCE(sio.tidx_blks_hit, 0),
+			 c.relpages,
+			 c.reltuples::bigint,
```

Comment:
`c.reltuples` is `float4`, but I'd say that we don't need that much detailed data (please correct me if I'm wrong), so I decided to go with `bigint` (aka `int64` in golang/protobuf). Other two fields are `int4` so it should be fine with `int32` in golang/protobuf.

Reply:
I suspect the motivation on the Postgres side here was to maybe save a few bytes (i.e. use 4 instead of 8 bytes) but still be able to approximate larger values. That's probably a fair trade-off in Postgres itself, since reltuples is an estimate anyway (not an accurate count of all tuples).

I think its acceptable to turn this into an int64 for ease of handling, since space is not as much a premium for us in the snapshots.

Reply:
ah, uhmm you made me realize that I can keep this as `float4` (32 bit float; `float4` or `real` in pg, `float` in protobuf, `float32` in golang, `f32` in Rust), so I'll go with that